newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/UserSurveyQuestion.js

```diff
@@ -0,0 +1,421 @@
+import Text from '../../../../UI/Text';
+import TextField from '../../../../UI/TextField';
+import Paper from '../../../../UI/Paper';
+import InlineCheckbox from '../../../../UI/InlineCheckbox';
+import RaisedButton from '../../../../UI/RaisedButton';
+import {
+  useResponsiveWindowWidth,
+  type WidthType,
+} from '../../../../UI/Reponsive/ResponsiveWindowMeasurer';
+import { ColumnStackLayout } from '../../../../UI/Layout';
+import { isOnlyOneFreeAnswerPossible } from './UserSurvey';
+import { type MessageDescriptor } from '../../../../Utils/i18n/MessageDescriptor.flow';
+
+const getColumnsFromWidth = (width: WidthType) => {
+  switch (width) {
+    case 'small':
+      return 1;
+    case 'medium':
+      return 2;
+    case 'large':
+      return 3;
+    case 'xlarge':
+    default:
+      return 4;
+  }
+};
+
+export const TitleAndSubtitle = ({
+  i18n,
+  text,
+  multi,
+  answers,
+  textAlign,
+}: {
+  i18n: I18nType,
+  text: MessageDescriptor,
+  multi: ?boolean,
+  answers: AnswerData[],
+  textAlign: 'center' | 'left',
+}) => (
+  <>
+    <Text size="block-title" align={textAlign}>
+      {i18n._(text)}
+    </Text>
+    {multi ? (
+      <Text style={styles.subTitle} align={textAlign}>
+        {i18n._(t`You can select more than one.`)}
+      </Text>
+    ) : isOnlyOneFreeAnswerPossible(answers) ? (
+      <Text style={styles.subTitle} align={textAlign}>
+        {i18n._(
+          t`The more descriptive you are, the better we can match the content we’ll recommend.`
+        )}
+      </Text>
+    ) : null}
+    <Spacer />
+  </>
+);
+
+const styles = {
+  answerButton: {
+    border: 'solid',
+    borderWidth: 1,
+    borderRadius: 8,
+    width: '100%',
+    height: '100%',
+    display: 'flex',
+  },
+  answerButtonBackground: { width: '100%', height: '100%' },
+  answerCoverImage: {
+    borderTopLeftRadius: 8,
+    borderTopRightRadius: 8,
+    width: '100%',
+  },
+  answerCheckboxAnchor: { position: 'relative' },
+  answerCheckboxContainer: {
+    position: 'absolute',
+    left: 5,
+    top: 'calc(50% - 9px)',
+  },
+  answerTextContainer: { marginLeft: 25, marginRight: 25 },
+  freeAnswerContent: {
+    display: 'flex',
+    height: '100%',
+    flexDirection: 'column',
+  },
+  freeAnswerInputOutline: {
+    border: 'solid',
+    borderWidth: 1,
+    borderRadius: 4,
+    flex: 1,
+    padding: 8,
+  },
+  subTitle: { opacity: 0.6 },
+};
+
+type FreeAnswerProps = {|
+  answerData: AnswerData,
+  onSelect: string => void,
+  selected: boolean,
+  i18n: I18nType,
+  showCheckbox: boolean,
+  onClickSend?: string => void,
+  value: string,
+  onChange: string => void,
+|};
+
+const FreeAnswer = ({
+  answerData,
+  i18n,
+  onSelect,
+  selected,
+  showCheckbox,
+  onClickSend,
+  value,
+  onChange,
+}: FreeAnswerProps) => {
+  const { text, imageSource, id } = answerData;
+  const [errorText, setErrorText] = React.useState<React.Node>(null);
+  const muiTheme = useTheme();
```

Comment:
is this on purpose? shouldn't this be `gdevelopTheme` ?

Reply:
Yes, that's different. There's the Material UI theme and the GDevelop theme. You can see this is taken from `@material-ui/styles`.

Reply:
ok, I was wondering if we couldn't use the gdevelop theme to get the same values, cause it felt to me this was the first time we used the MUI theme like that, but I may be wrong